calendar/page.js

```diff
@@ -310,6 +332,14 @@ async function updateCalendar(records, mappings) {
   dataVersion = Date.now();
 }
 
+function defaultFocus() {
```

Comment:
why "default"Focus? 
Form what i can see, this method just put focus on the window, so why not "focusOnWidget"?